rpcs3/Emu/RSX/VK/vkutils/instance.hpp

```diff
@@ -133,6 +138,27 @@ namespace vk
 
 			std::vector<const char*> extensions;
 			std::vector<const char*> layers;
+			const void* next_info = nullptr;
+
+#ifdef __APPLE__
```

Comment:
Move this into the "if !fast" block. We don't need to configure fast math or lost device behavior in that mode. Fast instances are not used to render, just enumerate devices and properties.

Reply:
but then i have 2 ifdef blocks due to the struct lifetime 😬

Reply:
done